MiniApp.podspec

```diff
@@ -12,14 +12,20 @@ Pod::Spec.new do |s|
   s.authors      = "Rakuten Ecosystem Mobile"
   s.summary      = "Rakuten's Mini App SDK"
   s.homepage     = "https://github.com/rakutentech/ios-miniapp"
-  s.license      = { :type => 'MIT', :file => 'LICENSE' }
-  s.source       = { :git => "https://github.com/rakutentech/ios-miniapp.git", :tag => 'v' + s.version.to_s }
+  s.license      = {
+    :type => 'MIT',
+    :file => 'LICENSE'
+  }
+  s.source       = {
+    :git => "https://github.com/rakutentech/ios-miniapp.git",
+    :tag => 'v' + s.version.to_s,
+  }
   s.documentation_url = "https://rakutentech.github.io/ios-miniapp/"
 
   s.ios.deployment_target = '11.0'
 
   s.source_files = 'MiniApp/Classes/**/*.swift'
-  s.resources = 'MiniApp/**/*.{xcassets,js,pdf,xib}'
+  s.resources = ['MiniApp/**/*.{xcassets,js,pdf,xib}','js-miniapp/js-miniapp-bridge/export/ios/bridge.js']
```

Comment:
You can update the submodule to use the latest commit on this branch: https://github.com/rakutentech/js-miniapp/tree/js-bridge-ios


And then the path for the JS file will be just `js-miniapp/bridge.js`.

Reply:
done